src/pocketmine/entity/Minecart.php

```diff
@@ -37,19 +38,36 @@ class Minecart extends Vehicle{
 	const TYPE_CHEST = 2;
 	const TYPE_HOPPER = 3;
 	const TYPE_TNT = 4;
+	
+	const STATE_INITIAL = 0;
+	const STATE_ON_RAIL = 1;
+	const STATE_OFF_RAIL = 2;
 
-	public $height = 0.9;
-	public $width = 1.1;
+	public $height = 0.7;
+	public $width = 0.98;
 
 	public $drag = 0.1;
 	public $gravity = 0.5;
 
 	public $isMoving = false;
 	public $moveSpeed = 0.4;
-
+	
+	private $onRail = MineCart::STATE_INITIAL;
```

Comment:
I would be happy to fix.  Are you saying that STATE_INITIAL should be StateInitial?  I was following what I saw in code for other constants like TYPE_HOPPER